src/Racing.cpp

```diff
@@ -14,35 +12,125 @@ std::map<int32_t, EPInfo> Racing::EPData;
 std::map<CNSocket*, EPRace> Racing::EPRaces;
 std::map<int32_t, std::pair<std::vector<int>, std::vector<int>>> Racing::EPRewards;
 
-static void racingStart(CNSocket* sock, CNPacketData* data) {
-    auto req = (sP_CL2FE_REQ_EP_RACE_START*)data->buf;
+namespace {
```

Comment:
Don't use a namespace to isolate the helpers, just make them static so they're local to the file. This is the pattern we use in the rest of the codebase.